### ISSUE TITLE
Support themes

### ISSUE THREAD

**Issue by smpallen99** [smpallen99/ex_admin#53]
Replace the activeadmin UI with a new UI skin. At that same time, allow for different add-on themes.

**Comment by smpallen99** [comment]
I have a new theme partially working. I'm thinking this will be the the default them, with the option of using the old theme. Note that the theme selector at the top right of the page can be disabled by configuration. Any feedback on the new theme?

<img width="1385" alt="screen shot 2016-04-24 at 9 43 52 am" src="https://cloud.githubusercontent.com/assets/891045/14767960/7ea716c6-0a01-11e6-9e7e-8d33b116a6e2.png">

<img width="1385" alt="screen shot 2016-04-24 at 9 40 16 am" src="https://cloud.githubusercontent.com/assets/891045/14767965/932f0a2c-0a01-11e6-984b-fc4a0d85ee48.png">

**Comment by OvermindDL1** [comment]
That looks very nice!  Looking forward!

**Comment by vinnymac** [comment]
@smpallen99 I like the new look 👍 

Any plans on using responsive styles akin to https://github.com/ball-hayden/responsive_active_admin ?

**Comment by smpallen99** [comment]
@vinnymac Like this?

I think the index table needs some work...

![screen shot 2016-04-26 at 12 45 33 pm](https://cloud.githubusercontent.com/assets/891045/14826703/1beb143a-0bad-11e6-9555-6f35aae9c0ef.png)
![screen shot 2016-04-26 at 12 46 07 pm](https://cloud.githubusercontent.com/assets/891045/14826704/1bec7866-0bad-11e6-8eba-2fecde6bfca7.png)
![screen shot 2016-04-26 at 12 46 29 pm](https://cloud.githubusercontent.com/assets/891045/14826706/1bf1b812-0bad-11e6-9647-f3b7e886b321.png)
![screen shot 2016-04-26 at 12 46 42 pm](https://cloud.githubusercontent.com/assets/891045/14826705/1beced46-0bad-11e6-96ce-8f34cdb087a8.png)

**Comment by smpallen99** [comment]
I just pushed the new template support to `themes` branch. Would be great if someone could give it a try and give me some feedback. Note, this is an ecto 1.x branch.

**Comment by smpallen99** [comment]
@sobolevn Could you give the new themes branch a try? Would be great to get your feedback. It should up up to day with the latest v0.7.3 branch.

**Comment by sobolevn** [comment]
@smpallen99 sure thing.

**Comment by sobolevn** [comment]
It does not work for me.

What have I done:
1. Removed everything from `_build`, `deps`, disabled cache in browser
2. Changed `mix`'s `deps` to: `{:ex_admin, git: "https://github.com/smpallen99/ex_admin.git", branch: "themes"},`
3. Changed the `config.exs`:

```
config :ex_admin,
  theme: ExAdmin.Theme.AdminLte2,
  theme_selector: [
    {"AdminLte",  ExAdmin.Theme.AdminLte2},
    {"ActiveAdmin", ExAdmin.Theme.ActiveAdmin}
  ],
  ...
```

4. `mix admin.install` does not even create `themes` folder, however I have added it to the `endpoint.ex`

I can't see any changes. I am doing something wrong?

**Comment by smpallen99** [comment]
@sobolevn I suspect your still picking up the old version. Try this:
```
mix deps.clean ex_admin
mix deps.get
mix deps.compile
```

**Comment by sobolevn** [comment]
I have managed to get it to work by changing the `mix.exs` to:

`{:ex_admin, github: "smpallen99/ex_admin", ref: "aba5379769f733a03a7756a72e0b7e2689e6e75d"},`

And `mix.lock` (was not working before that): 

`  "ex_admin": {:git, "https://github.com/smpallen99/ex_admin.git", "aba5379769f733a03a7756a72e0b7e2689e6e75d", [ref: "aba5379769f733a03a7756a72e0b7e2689e6e75d"]},`

It looks pretty nice! I can do a full browser-stack testing tomorrow. 
<img width="528" alt="2016-05-02 2 53 15" src="https://cloud.githubusercontent.com/assets/4660275/14945088/63b61f36-1011-11e6-97df-00adec800f98.png">

**Comment by smpallen99** [comment]
Just updated [ExAdmin Demo](http://demo.exadmin.info/admin/products) with the new theme.

**Comment by sobolevn** [comment]
It looks pretty solid across all devices I was able to test. Did not find any bugs. Here are some screenshots:

<img width="456" alt="2016-05-02 14 27 06" src="https://cloud.githubusercontent.com/assets/4660275/14953419/0e37201c-1072-11e6-954b-935d0e813f7e.png">
<img width="458" alt="2016-05-02 14 26 45" src="https://cloud.githubusercontent.com/assets/4660275/14953420/0e393bea-1072-11e6-912e-6088e1daa6fb.png">
<img width="413" alt="2016-05-02 14 25 56" src="https://cloud.githubusercontent.com/assets/4660275/14953421/0e3a4cce-1072-11e6-988e-970e6bfc267c.png">
<img width="480" alt="2016-05-02 14 25 23" src="https://cloud.githubusercontent.com/assets/4660275/14953422/0e3fa570-1072-11e6-8de7-fc4b0b7ff3dd.png">

**Comment by smpallen99** [comment]
Just published 0.7.4 to Hex with this feature. Content also in master for ecto 2.0.
